manifests/init.pp

```diff
@@ -146,6 +146,14 @@
       }
     }
 
+    file {'/etc/httpd/conf.d/ssl.conf':
+      ensure  => file,
+      content => '# Managed by Puppet',
```

Comment:
@ehelms ack will add this and resubmit once stbenjam finishes with his testing and pulls in the pr I put in yesterday adding the env variable so the tests will go through with this pr.